web/src/frontend/classic/components/content-pages/Documentation.svelte

```diff
@@ -0,0 +1,94 @@
+<script lang="ts">
+    import {
+        Tile,
+        ClickableTile,
+        Breadcrumb,
+        BreadcrumbItem,
+    } from 'carbon-components-svelte';
+    import {
+        documentation,
+        type DocCategory,
+        type DocSection,
+    } from '../../stores/Documentation';
+    let selectedDocSection: DocSection;
+    let selectedDocCategory: DocCategory;
+</script>
+
+{#if selectedDocSection}
+    <Breadcrumb noTrailingSlash>
+        <BreadcrumbItem
+            href="#"
+            on:click={() => {
+                selectedDocSection = null;
+                selectedDocCategory = null;
+            }}>Documentation</BreadcrumbItem
+        >
+        <BreadcrumbItem>{selectedDocCategory?.name}</BreadcrumbItem>
+        <BreadcrumbItem isCurrentPage>{selectedDocSection?.name}</BreadcrumbItem
+        >
+    </Breadcrumb>
+    <div class="article">{@html selectedDocSection?.content.outerHTML}</div>
+{:else}
+    {#each $documentation as docCategory}
+        <div class="category">
+            <h3 class="name">{docCategory.name}</h3>
+            <div class="content">
+                {@html docCategory.description.outerHTML}
```

Comment:
Use the script to attach the `docCategory.description` sub-tree as child nodes into the `div` container and avoid injecting HTML as string.

Reply:
this is the svelte way to embed html as @html is a special tag
https://svelte.dev/docs/special-tags#html

however this should do the trick
`<div bind:this={docCategory.description}>  `

Reply:
trick cannot be used as the binding  requires a set function on description.
I'll stick to @html for now as rows are generated using #each